.eslintignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Is this file strictly necessary for the change to work? See [The .eslintignore File](https://eslint.org/docs/latest/use/configure/ignore#the-eslintignore-file)

> In addition to any patterns in the .eslintignore file, ESLint always follows a couple of implicit ignore rules even if the --no-ignore flag is passed. The implicit rules are as follows:
> - node_modules/ is ignored.
> - dot-files (except for .eslintrc.*) as well as dot-folders and their contents are ignored.

Reply:
It really is no longer necessary to ignore the `node_modules` folder manually. The `eslint` seems to be doing this by default since 2014. https://github.com/btmills/eslint/commit/f0be1accf251f6acd2066bfc90705f4670e2146e

But most people don't know that. You can see this being done even in large projects. https://github.com/vercel/next.js/blob/canary/.eslintignore
https://github.com/webpack/webpack/blob/main/.eslintignore
https://github.com/mui/material-ui/blob/master/.eslintignore

Reply:
We are fine then without having it at current code base.
Thanks for the clarification!